3-networks/README.md

```diff
@@ -16,114 +65,197 @@ The purpose of this step is to:
 1. After production has been applied, apply development and non-production.
-1. Merge changes to development with `git checkout -b development` and `git push origin development`.
-    1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
-1. Merge changes to non-production with `git checkout -b non-production` and `git push origin non-production`.
-    1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to development
+   ```
+   git checkout -b development
+   git push origin development
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to non-production.
+   ```
+   git checkout -b non-production
+   git push origin non-production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
 
 1. You can now move to the instructions in the step [4-projects](../4-projects/README.md).
 
-### Run terraform locally
+### Deplying with Cloud Build
+
+1. Clone repo.
+   ```
+   gcloud source repos clone gcp-networks --project=YOUR_CLOUD_BUILD_PROJECT_ID
+   ```
+1. Change to the freshly cloned repo and change to non-master branch.
+   ```
+   git checkout -b plan
+   ```
+1. Copy contents of foundation to new repo.
+   ```
+   cp -RT ../terraform-example-foundation/3-networks/ .
+   ```
+1. Copy cloud build configuration files for Terraform.
+   ```
+   cp ../terraform-example-foundation/build/cloudbuild-tf-* .
+   ```
+1. Copy terraform wrapper script to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/tf-wrapper.sh .
+   ```
+1. Ensure wrapper script can be executed.
+   ```
+   chmod 755 ./tf-wrapper.sh
+   ```
+1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap. See any of the envs folder [README.md](./envs/production/README.md) files for additional information on the values in the `common.auto.tfvars` file.
+1. Rename `shared.auto.example.tfvars` to `shared.auto.tfvars` and update the file with the target_name_server_addresses (the list of target name servers for the DNS forwarding zone in the DNS Hub).
+1. Rename `access_context.auto.example.tfvars` to `access_context.auto.tfvars` and update the file with the `access_context_manager_policy_id`.
+1. Commit changes
+   ```
+   git add .
+   git commit -m 'Your message'
+   ```
+1. You need to manually plan and apply only once the `shared` environment since `development`, `non-production` and `production` depend on it.
```

Comment:
```suggestion
1. You need to manually plan and apply only once the `shared` environment since `development`, `non-production`, and `production` depend on it.
```